src/modules/comments.js

```diff
@@ -0,0 +1,149 @@
+const getCounter = (data) => (typeof (data) === 'object' ? data.length : 'invalid');
+
+const commentsApiKey = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XAL29eekekrMc3kTORyS/comments';
+const getComment = async (movieId) => {
+  const response = await fetch(`${commentsApiKey}?item_id=${movieId}`);
+  return response.json();
+};
+
+const commentPopup = document.querySelector('.comment-popup');
+const getAllComments = async (movieId) => {
+  const result = await getComment(movieId)
+    .then((comment) => (!comment.error ? comment.length : 0))
+    .catch(() => 0);
+  return result;
+};
```

Comment:
-  In Es6 the more you use modules the better, Kindly put this function in a module.
Then import it in the ```comment.js```
```bash
import getAllComments from './comments.js'
```
And export it in the created module using the code below :
```bash
export default getAllComments;
```